### PR TITLE
amends path to vale ini and covers metamask docs structure

### PR DESCRIPTION
**Pull request by m4sterbunny**

### CODE REVIEW

File: spelling/action.yml [hunk 1]
```diff
@@ -20,8 +20,14 @@ runs:
...
-        files: '["./docs/", "./blog/"]' 
-        vale_flags: "--config ./.docs-gha/spelling/config/.vale.ini --glob='!docs/**/*.js'"        
+        files: |
+          "./docs/", 
+          "./blog/", 
+          "./services/", 
+          "./developer-tools/", 
+          "./snaps/", 
+          "./wallet/" 
+        vale_flags: "--config ./.docs-gha/spelling/.vale.ini --glob='!docs/**/*.js'"
```
Comment (m4sterbunny) [inline spelling/action.yml:30]: @joshuafernandes I may have misread vale docs, but AFAIK the config folder is for vale-related config, not for the actual vale.ini -- I moved it up a level when fixing/testing

File: spelling/action.yml [hunk 2]
```diff
@@ -20,8 +20,14 @@ runs:
     - name: Vale
       uses: errata-ai/vale-action@v2
       with:
-        files: '["./docs/", "./blog/"]' 
-        vale_flags: "--config ./.docs-gha/spelling/config/.vale.ini --glob='!docs/**/*.js'"        
+        files: |
```
[the remark this answers is not in the document]
Comment (joshuafernandes) [reply spelling/action.yml:23]: Jolly good, do we only use this one for MM? If so can merge here, else I'll make another action or arg to do this optionally

File: spelling/action.yml (hunk 1 above)
Comment (joshuafernandes) [reply spelling/action.yml:30]: This will work well too, tbh prob better coz its a lot more explicit as to where to get it from

**Review by joshuafernandes — APPROVED** [review]
LGTM, one minor question re end use case, but can merge and sort that out later as well

File: spelling/action.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (m4sterbunny) [reply spelling/action.yml:23]: Not only MM, this is supposed to be ssot for all docs sites that want to join in on Vale linter. I was just trying to cover MM case (where docs aren't in docs folder!) as well as other cases while I remembered!

**Comment by m4sterbunny** [comment]
Thanks @joshuafernandes I don't have write access here to merge

**Pull request merged**